src/main/kotlin/org.hiero.gradle.feature.publish-maven-central-aggregation.gradle.kts

```diff
@@ -28,7 +28,7 @@ nmcpAggregation {
         username = providers.environmentVariable("NEXUS_USERNAME")
         password = providers.environmentVariable("NEXUS_PASSWORD")
         publishingType = if (publishTestRelease) "USER_MANAGED" else "AUTOMATIC"
-        validationTimeout = Duration.of(30, ChronoUnit.MINUTES)
+        validationTimeout = Duration.of(60, ChronoUnit.MINUTES)
```

Comment:
Maybe directly introduce a parameter as well so that this can be controlled in the pipelines without publishing new plugin updates?

See: https://github.com/hiero-ledger/hiero-gradle-conventions/issues/245#issuecomment-3049626833